src/framework/theme/components/button-toggle/_button-toggle.component.theme.scss

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+@import '../button/button-mixins';
+
+@mixin nb-buttons-toggle-theme() {
+  [nbButtonToggle] {
+    @extend [nbButton];
+  }
+
+  [nbButtonToggle].appearance-filled {
```

Comment:
Also need to change color of `[nbButton]`.

Reply:
![image](https://user-images.githubusercontent.com/22653273/95241250-294ac200-0816-11eb-9110-fadb99d254af.png)
This code was introduced to fix issue with toggle-buttons shown on attached image. I'm not sure why it happens, but i believe [nbButton] do not need to use this fix